### PR TITLE
Correct syntax for copying files from static directory

### PR DESCRIPTION
**Pull request by jingman**

### CODE REVIEW

**Comment by LinusBorg** [comment]
You seem to have mixed a lot of other changes into this PR, like an update to webpack 2 etc. 

We can't merge this in this state.

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,27 @@
+0 info it worked if it ends with ok
```
Comment (LinusBorg) [inline npm-debug.log]: this file belong in this PR

File: template/.eslintignore [hunk 2]
```diff
@@ -1,2 +0,0 @@
-build/*.js
```
Comment (LinusBorg) [inline template/.eslintignore]: Why?

File: template/.eslintrc.js [hunk 3]
```diff
@@ -1,46 +0,0 @@
-module.exports = {
```
Comment (LinusBorg) [inline template/.eslintrc.js]: ...and this shouldn't be deleted either.

There are many other changes that don'T belong here, and I will not comment on all of them.

**Comment by jingman** [comment]
Woah my bad, I've been iterating on this myself in the month since I opened the PR, and I wasn't' aware it was spamming you with updates. Sorry to waste your time - thank you for your work on Vue.

**Comment by LinusBorg** [comment]
it's all good. If you still can reproduce the original fix, even as a new PR, I'm, happy to merge it.

**Pull request closed without merging**